corsheaders/middleware.py

```diff
@@ -103,7 +103,9 @@ def process_response(self, request, response):
         Add the respective CORS headers
         """
         origin = request.META.get('HTTP_ORIGIN')
-        if request._cors_enabled and origin:
+        enabled = getattr(request, '_cors_enabled', None)
+        enabled = enabled if enabled is not None else self.is_enabled(request)
```

Comment:
> cleaner as

Needed an extra test (one that calls process_response directly (not via test client)) to get 100% coverage under django 1.10+. But it's done now.

>  also you can put everything underneath the if origin to avoid processing if origin is not set.

Not really what this PR was about in the first place. But also done now!